integration_tests/e2e/refer/show.cy.ts

```diff
@@ -1,211 +1,36 @@
-  const coursePresenter = CourseUtils.presentCourse(course)
-  const courseOffering = courseOfferingFactory.build()
-  const prison = prisonFactory.build({ prisonId: courseOffering.organisationId })
-  const prisoner = prisonerFactory.build({
-    dateOfBirth: '1980-01-01',
-    firstName: 'Del',
-    lastName: 'Hatton',
-    sentenceStartDate: '2010-10-31',
-  })
-  const person = personFactory.build({
-    currentPrison: prisoner.prisonName,
-    dateOfBirth: '1 January 1980',
-    ethnicity: prisoner.ethnicity,
-    gender: prisoner.gender,
-    name: 'Del Hatton',
-    prisonNumber: prisoner.prisonerNumber,
-    religionOrBelief: prisoner.religion,
-    sentenceStartDate: '2010-10-31',
-    setting: 'Custody',
-  })
-  const referral = referralFactory.submitted().build({
-    offeringId: courseOffering.id,
-    prisonNumber: person.prisonNumber,
-  })
-  const organisation = OrganisationUtils.organisationFromPrison(prison)
-
-  const user = userFactory.build()
-  const courseParticipationPresenter1 = {
-    ...courseParticipationFactory.build({
-      addedBy: user.username,
-      courseName: course.name,
-      prisonNumber: person.prisonNumber,
-    }),
-    addedByDisplayName: StringUtils.convertToTitleCase(user.name),
-  }
-  const courseParticipationPresenter2 = {
-    ...courseParticipationFactory.build({
-      addedBy: user.username,
-      courseName: 'Another course name',
-      prisonNumber: person.prisonNumber,
-    }),
-    addedByDisplayName: StringUtils.convertToTitleCase(user.name),
-  }
-
-  beforeEach(() => {
-    cy.task('reset')
-    cy.task('stubSignIn', { authorities: [ApplicationRoles.ACP_REFERRER] })
-    cy.task('stubAuthUser')
-    cy.task('stubDefaultCaseloads')
-    cy.signIn()
-
-    cy.task('stubCourseByOffering', { course, courseOfferingId: courseOffering.id })
-    cy.task('stubOffering', { courseId: course.id, courseOffering })
-    cy.task('stubPrison', prison)
-    cy.task('stubPrisoner', prisoner)
-    cy.task('stubReferral', referral)
-  })
-
   describe('When reviewing personal details', () => {
     it('shows the correct information', () => {
-      const path = referPaths.show.personalDetails({ referralId: referral.id })
-      cy.visit(path)
-
-      const personalDetailsPage = Page.verifyOnPage(PersonalDetailsPage, {
-        course,
-        person,
-      })
-      personalDetailsPage.shouldHavePersonDetails(person)
-      personalDetailsPage.shouldContainNavigation(path)
-      personalDetailsPage.shouldContainBackLink('#')
-      personalDetailsPage.shouldContainCourseOfferingSummaryList(coursePresenter, organisation.name)
-      personalDetailsPage.shouldContainSubmissionSummaryList(referral)
-      personalDetailsPage.shouldContainSubmittedReferralSideNavigation(path, referral.id)
-      personalDetailsPage.shouldContainImportedFromText()
-      personalDetailsPage.shouldContainPersonalDetailsSummaryCard()
+      sharedTests.referrals.show.personalDetails(ApplicationRoles.ACP_REFERRER)
```

Comment:
Can we make this something like `sharedTests.referrals.showsPersonalDetailsPage` to keep this in an assertion-style format?